.cargo/config

```diff
@@ -1,3 +1,11 @@
 [build]
 # Instruction set of Cortex-M0+
 target = "thumbv6m-none-eabi"
+
+[target.'cfg(all(target_arch = "arm", target_os = "none"))']
```

Comment:
Why we need those flags?

Reply:
They seem to be the magic incantations needed to get the embedded Rust stuff to build. I copied them from the sample project.

Reply:
`link-arg=--nmagic` is needed if flash or ram is not aligned to 0x10000

`link-arg=-Tlink.x"` is LLD linker (I think needed if you need to link C dependencies)

but i'm not sure why we need the last 2